kythe/cxx/indexer/textproto/analyzer.cc

```diff
@@ -631,6 +701,21 @@ absl::optional<std::string> ParseProtoMessageArg(
   return absl::nullopt;
 }
 
+absl::optional<std::vector<std::string>> ParsePluginArg(
+    std::vector<std::string>* args) {
+  for (size_t i = 0; i < args->size(); i++) {
```

Comment:
I would use iterators rather than indices as this will likely make the resulting code easier to read:
```
for (auto iter = args->begin(); iter != args->end(); iter++) {
  if (*iter == "--kythe_textproto_plugins") {
    if (iter + 1 < args->end()) {
      std::string value = *(iter + 1);
      args->erase(iter, iter + 2);
      return absl::StrSplit(value, ",");
    }
  }
}
```